### PR TITLE
fix opening sentence

### PR DESCRIPTION
**Pull request by jhdalek55**
first of several more changes.

### CODE REVIEW

File: adoptions.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ css_id: adoptions
 
 ### Adoptions
 
-(Many of the images below link to articles that cover the TUF adoption.)
+By clicking of the images below, you can link to articles that cover the TUF
```
Comment (vladimir-v-diaz) [inline adoptions.md]: "By clicking of the images below, ..."

Is there a typo here?

File: news.md [hunk 2]
```diff
@@ -66,32 +66,32 @@ please see [TUF Augmentation Proposals
...
 on TUF and Notary at LinuxCon North America.  Slides of his talk are available
-here:
-https://events.linuxfoundation.org/events/linuxcon-north-america/program/slides
+[here](https://events.linuxfoundation.org/events/linuxcon-north-america/program/slides).
+
+**March 18, 2016**
+
+> 
+Trishank Kuppusamy presents "Diplomat: Using Delegations to Protect Community
+Repositories" at [NSDI 2016](https://www.usenix.org/conference/nsdi16).
```
Comment (vladimir-v-diaz) [inline news.md]: Should we link to the page that contains the presentation slides, audio recording, paper, etc.?

https://www.usenix.org/node/194973

File: overview.md [hunk 3]
```diff
@@ -21,8 +21,7 @@ growing flow of updates has also created a need for better
 ways to protect the systems that manage them. Though a number of strategies have
 been introduced and used over the last decade or so to enhance the
 authenticity—and by extension, the security of update systems—most have drawbacks
-that have left update files and the repositories that host them.
-vulnerable to a number of attacks.
+that have left update files and the repositories that host them s,vulnerable to a number of attacks.
```
Comment (vladimir-v-diaz) [inline overview.md]: typo: " ... that host them s, ..."

File: overview.md [hunk 4]
```diff
@@ -73,18 +72,18 @@ account, such as when:
   there is an update.
 
 * An attacker gives you an older, insecure version of a file that you
-  already have, so you download it and blindly use it thinking it's
-  newer.
+  already have and tricks you into thinking it's
+  newer. You download it an dblindly use it.
```
Comment (vladimir-v-diaz) [inline overview.md]: typo: "... dblindly ..."

**Comment by jhdalek55** [comment]
Yes, in adoptions that was a typo--should be clicking "on." I usually print
a hardcopy to proof before finalizing. With the printer being out of toner
indefinitely, I had to proof onscreen only. I proofread better when I have
it on paper in front of me.

I'll clean up the other typos you found. Thanks for giving it a careful
read.

On the News section...yeah...I don't think it would hurt to link to the
slides or the video there..

Lois

**Comment by jhdalek55** [comment]
Two more things I noticed when I was checking through the web site for data
for a revision on Wikipedia...First off, we mentioned the two PEPs but I
could find no link to these two documents. If we don't want to link to
these texts through the publications list, can we embed a link on the
Timeline where they are mentioned?

Second, on the new publications page, the links for the last two items: A
Look In the Mirror: Attacks on Package Managers
<https://theupdateframework.github.io/papers/package-management-security-tr08-02.pdf?raw=true>
Package Management Security
<https://theupdateframework.github.io/papers/attacks-on-package-managers-ccs2008.pdf?raw=true>
  are reversed.

I can fix these when I go in to correct the other typos you found if you
let me know where you want the PEPs.

Lois

**Comment by vladimir-v-diaz** [comment]
The PEP can be found here:
https://www.python.org/dev/peps/pep-0458/
https://www.python.org/dev/peps/pep-0480/

There isn't a single document for each PEP, so we'll have to link to them from our website.  More recent versions of the PEPs are under revision control on the interoperability repo.
https://github.com/pypa/interoperability-peps/blob/master/pep-0458-tuf-online-keys.rst

**Comment by jhdalek55** [comment]
I honestly don't remember but I'm pretty sure I didn't address your 
comments. I've been pretty deeply immersed in another project the last 
two weeks. I will look on Monday and address all your comments then.


Thanks for the reminder and I apologize for the delay.


Lois
On Fri, Apr 13, 2018 at 02:18 PM, Vladimir Diaz wrote:

@jhdalek55 <https://github.com/jhdalek55>  Is this a work in progress? 
I'm not sure if you addressed my comments (and haven't pushed them yet), 
or you still need more time.


—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub 
<https://github.com/theupdateframework/theupdateframework.github.io/pull/32#issuecomment-381220325> 
, or mute the thread

**Comment by vladimir-v-diaz** [comment]
Thanks!  Looks good to me, I only made a small change to the headings.

**Pull request merged**